aim/db/migration/alembic_migrations/versions/5e29ae4f45e6_host_domain_add_primary_key.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2017 Cisco Systems
```

Comment:
should make the file name more clear like 5e29ae4f45e6_host_domains_mapping_v2.py as we are no longer adding primary keys to the old table.

Reply:
good point -- will rename.